crypto/x509/pcy_node.c

```diff
@@ -113,6 +116,13 @@ X509_POLICY_NODE *ossl_policy_level_add_node(X509_POLICY_LEVEL *level,
     return node;
 
  node_error:
+    if (level != NULL && node_stored) {
+        if (OBJ_obj2nid(data->valid_policy) == NID_any_policy) {
+            level->anyPolicy = NULL;
+        } else {
```

Comment:
the braces around single statements could go away here.

Reply:
Done.